flow/e2e/snowflake/peer_flow_sf_test.go

```diff
@@ -5,40 +5,76 @@ import (
 	"fmt"
 	"strings"
 	"testing"
+	"time"
 
 	connsnowflake "github.com/PeerDB-io/peer-flow/connectors/snowflake"
 	"github.com/PeerDB-io/peer-flow/e2e"
 	"github.com/PeerDB-io/peer-flow/generated/protos"
 	"github.com/PeerDB-io/peer-flow/model/qvalue"
+	util "github.com/PeerDB-io/peer-flow/utils"
 	peerflow "github.com/PeerDB-io/peer-flow/workflows"
 	"github.com/jackc/pgx/v5/pgxpool"
 	"github.com/joho/godotenv"
 	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"go.temporal.io/sdk/testsuite"
 )
 
-const snowflakeSuffix = "snowflake"
-
 type PeerFlowE2ETestSuiteSF struct {
 	suite.Suite
 	testsuite.WorkflowTestSuite
 
+	pgSuffix  string
 	pool      *pgxpool.Pool
 	sfHelper  *SnowflakeTestHelper
 	connector *connsnowflake.SnowflakeConnector
 }
 
 func TestPeerFlowE2ETestSuiteSF(t *testing.T) {
-	suite.Run(t, new(PeerFlowE2ETestSuiteSF))
+	s := &PeerFlowE2ETestSuiteSF{}
+	s.SetT(t)
+	s.SetupSuite()
+
+	tests := []struct {
```

Comment:
#629 added code to `test_utils.go` to generate this kind of table automatically with `reflect`